code/modules/paperwork/paperbin.dm

```diff
@@ -120,50 +144,92 @@
 
 /obj/item/paper_bin/update_overlays()
 	. = ..()
+
+	total_paper = LAZYLEN(papers)
+
+	if(bin_pen)
+		pen_overlay = mutable_appearance(bin_pen.icon, bin_pen.icon_state)
+
+	if(!bin_overlay)
+		bin_overlay = mutable_appearance(icon, bin_overlay_string)
+
+	if(LAZYLEN(papers))
+		for(var/paper_number in 1 to LAZYLEN(papers))
+			if(paper_number != LAZYLEN(papers) && paper_number % 8 != 0) //only top paper and every 8th paper get overlays
+				continue
+			var/obj/item/paper/current_paper = papers[paper_number]
+			var/mutable_appearance/paper_overlay = mutable_appearance(current_paper.icon, current_paper.icon_state)
+			paper_overlay.color = current_paper.color
+			paper_overlay.pixel_y = paper_number/8 - 2 //gives the illusion of stacking
```

Comment:
Where's the 2 come from?

Reply:
The paper overlays need to be shifted down 2 pixels so they are in the right position.